basket/base/views.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.admin.views.decorators import staff_member_required
+from django.shortcuts import render
+
+from basket.base.forms import EmailListForm
+from basket.news.backends.ctms import (
+    CTMSNotFoundByEmailError,
+    ctms,
+)
+
+
+@staff_member_required
```

Comment:
I'm thinking it might be nice to also make this require certain permissions so not everyone with a Basket login sees this.

Reply:
Added the DSAR user permission and group to the 2nd commit on this PR.